### PR TITLE
removed padding from subtext to align with header

### PR DESCRIPTION
**Pull request by giladt**
- Padding to the left was removed.
- Added top padding to align sub description to the center of the button. 

Closes this [issue](https://www.notion.so/curvelabs/Check-Eligibility-should-be-aligned-with-Subtext-5272a87913ea4c19a809713ec2ee652d)

### CODE REVIEW

File: src/seedDashboard/seedDashboard.scss [hunk 1]
```diff
@@ -278,9 +278,8 @@
           justify-self: flex-start;
         }
         .subdescription {
-          padding-left: 29px;
+          padding: 16px 0px;
```
Comment (dkent600) [inline src/seedDashboard/seedDashboard.scss:281]: Best to make only the minimal change necessary.  Can you please just change padding-left?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/seedDashboard/seedDashboard.scss (hunk 1 above)
Comment (giladt) [reply src/seedDashboard/seedDashboard.scss]: I needed not only to change the left padding but also add top padding to the `.subDescription` class, it makes the code more readable in my opinion when is shorter and in general more performant.

File: src/seedDashboard/seedDashboard.scss [hunk 2]
```diff
@@ -278,8 +278,9 @@
           justify-self: flex-start;
         }
         .subdescription {
-          padding-left: 29px;
+          padding-left: 0px;
           padding-right: 29px;
+          padding-top: 16px;
```
Comment (dkent600) [inline src/seedDashboard/seedDashboard.scss:283]: Why did you add this?  It wasn't called for in the card

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/seedDashboard/seedDashboard.scss (hunk 1 above)
Comment (dkent600) [reply src/seedDashboard/seedDashboard.scss]: The card didn't ask for a change in the top padding did it?

File: src/seedDashboard/seedDashboard.scss (hunk 2 above)
Comment (giladt) [reply src/seedDashboard/seedDashboard.scss:283]: I have discussed this with @BartuCulha earlier this morning and got his consent to do that.

Comment (giladt) [reply src/seedDashboard/seedDashboard.scss:283]: I've added an image and a comment for that in the card, but as I checked it as resolved, now the comment is hidden 😕

**Review by dkent600 — APPROVED** [review]
I removed the vertical padding change because of consensus between Renc and me.  And removed a period that didn't belong.
